src/oas2/transformers/params.ts

```diff
@@ -81,41 +82,84 @@ export function translateToBodyParameter(body: BodyParameter, consumes: string[]
 }
 
 export function translateFromFormDataParameter(
-  form: FormDataParameter,
-  requestBody: IHttpOperationRequestBody | null | undefined,
+  parameters: FormDataParameter[],
   consumes: string[],
 ): IHttpOperationRequestBody {
-  const baseContent = buildContentFromParameter(form);
```

Comment:
Apparently you've changed this function quite a lot. According to the code this was transforming a *single* parameter, now it's transforming all the parameters all together. Was there any particular reason for this?

Reply:
Hey, good question. Several reasons:
1. The original function horrible - it "kind of mutated inputs"...
2. An array of parameters is reduced into a single schema - it previously took each param individually and "existing body" in order to "mutate" that existing body. That was redundant and confusing.
3. It didn't build `encodings` correctly.

Reply:
In that case we should change the name